package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bc-registry",
-  "version": "2.0.4",
+  "version": "2.1.0",
```

Comment:
major version upgrade and nothing has changed other than CI/CD :(

it's fine

Reply:
That's a minor version change and files were deleted.